problemsets-mat223/linearalbegra.tex

```diff
@@ -2974,24 +3010,51 @@ \section*{Linear Transformations}
 		Let $L:V\to W$ be a transformation and let $X\subset V$ be a set.
 		The \emph{image of the set $V$ under $L$}, denoted $L(V)$, is the set
 		\[
-			L(V)=\{\vec x: \vec x=L(\vec y)\text{ for some }\vec y\in V\}.
+			L(V)=\Set{\vec x\in W \given \vec x=L(\vec y)\text{ for some }\vec y\in V}.
 		\]
 	\end{definition}
 
 	\question
-	Let $S=\{\mat{x\\y}:0\leq x,y\leq 1\}\subseteq \R^2$ be the filled-in unit square
-	and let $C=\{\vec 0, \vec e_1,\vec e_2,\vec e_1+\vec e_2\}\subseteq \R^2$ be the corners of the 
-	unit square.
+	Let $S=\Set*{\mat{x\\y} \given 0\leq x,y\leq 1}\subseteq \R^2$ be the filled-in unit 
+	square and let $C=\Set{\vec 0, \vec e_1,\vec e_2,\vec e_1+\vec e_2}\subseteq \R^2$
+	be the corners of the unit square.
 	\begin{parts}
 		\item Find $\mathcal R(C)$, $W(C)$, and $T(C)$ (where $\mathcal R$, $W$,
 			and $T$ are from the previous question).
-		\item Draw $\mathcal R(S)$, $T(S)$, and $\mathcal P(S)$ (where $\mathcal R$, $T$,
-			and $\mathcal P$ are from the previous question).
-		\item Suppose that $\ell=\{\text{ all convex combinations of }\vec a\text{ and }\vec b\}$
-			is a line segment with endpoints $\vec a$ and $\vec b$
-			and $A$ is a linear transformation. Must $A(\ell)$ be a line segment? What are
-			its endpoints?
-		\item Explain how images of sets relate to the \emph{Italicising N} task.
+			\begin{solution}
+				$\mathcal R(C) = \Set{\vec 0,\vec e_2,-\vec e_1,-\vec e_1+\vec e_2}$.
+
+				$W(C) = C$.
+
+				$T(C) = \Set*{\mat{2\\0},\mat{3\\0},\mat{2\\1},\mat{3\\1}}$.
+			\end{solution}
+		\item Draw $\mathcal R(S)$, $T(S)$, and $\mathcal P(S)$ (where $\mathcal R$,
+			$T$, and $\mathcal P$ are from the previous question).
+			\begin{solution}
+				XXXX
+			\end{solution}
+		\item Suppose that $\ell=\Set{\text{all convex combinations of }\vec a\text{ and }\vec b}$
```

Comment:
Rephrase: Let \ell ... be the line segment and let A be a linear transformation.

Reply:
Not entirely sure what you mean here. The problem defines \ell and A so I ordinarily wouldn't say "Let..." again. But are you imagining something like this:

Let \ell = {all convex linear combinations of a and b} be the line segment and A be a linear transformation. By the linearity of A

A(\alpha_1 a + \alpha_2 b) = \alpha_1 A(a) + \alpha_2 A(b).

If \alpha_1 + \alpha_2 = 1, then the linear combination on the right is also convex, and so A(\ell) is the set of convex combinations of A(a) and A(b). This is precisely the straight line segment joining A(a) and A(b).

Reply:
Yep! That's what I was thinking. Though, I meant to suggest that you rephrase the problem as "Let ...". You don't need to say "let" in your solution.